src/features/student/modules/Rating.module.scss

```
@@ -0,0 +1,30 @@
+@import '../../../colors.module.scss';
+
+.ratingContainer {
+    color: black;
+    font-weight: bold;
+    display: flex;
+    justify-content: center;
+    margin: 0 10px 0 10px;
+
+    .rating {
+        border-radius: 5px;
+        padding: 2px 10px 2px 10px;
```

```suggestion
padding: 2px 10px;
```
First value is vertical, second horizontal padding